runtime/lua/vim/lsp/protocol.lua

```diff
@@ -621,6 +621,56 @@ export interface WorkspaceClientCapabilities {
 function protocol.make_client_capabilities()
   return {
     textDocument = {
+      semanticTokens = {
+        dynamicRegistration = false;
+        tokenTypes = {
+          'namespace',
```

Comment:
Can we keep these in the same order as the protocol?

Reply:
sure!.